front/app/containers/Admin/projects/components/ProjectRow/AdminTag.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { colors } from 'utils/styleUtils';
+import { StyledStatusLabel } from '.';
+import ConditionalWrapper from 'components/ConditionalWrapper';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+import Link from 'utils/cl-router/Link';
+import { adminProjectsProjectPath } from 'containers/Admin/projects/routes';
+
+interface Props {
+  userCanModerateProject: boolean;
+  projectId: string;
+}
+
+const AdminTag = ({ userCanModerateProject, projectId }: Props) => {
+  return (
+    <ConditionalWrapper
```

Comment:
I see now how this is used. Let's maybe remove it - if we insist on doing this with components, we already have a `Permission` component that can serve this purpose. But it's best to use our React hook, I think.

Reply:
The most common use case is wrapping something with the Link component if a condition is met to avoid messy/repetitive returns/render functions. You can also see it in e.g. front/app/components/Avatar/index.tsx. 

It's about not about permissions per se. The condition specifically is not important. So how would you do it then with my 2 components?

Reply:
Maybe it's a bit subjective but I find this pattern a bit confusing and harder to read than the alternative. I would stick with explicitly returning the link under a certain condition and the component in the other case. I would, however put the link return on top to make it more clear what is happening.

Reply:
I've had this experience before with hooks vs. classes and lifecycle methods, with calling functions like onChange?.() instead of onChange && onChange(), with myVar ?? null, etc. New concepts always take some getting used to.

Personally, I find it pretty neat compared to the alternative. :-)

Reply:
Shall we let the FE team vote? 😁

Reply:
Removed it for now.